### DISCUSSION TITLE
scrollwheel=false

### DISCUSSION THREAD

**Discussion by sfx12** [Tastaturberuf/anyStores#94] [Feature Requests / Funktionswünsche]
wo setze ich die option, damit man in die map nicht zoomen kann mit dem scrollrad der maus ?

bei fullwidth layouts...ziemlich nervig beim scrollen der website !!

Danke schon mal im vorraus

**Comment by Tastaturberuf** [comment]
Momentan nicht möglich. Nicht updatesicher kann die anystores.js über
schrieben werden.

**Comment by sfx12** [comment]
ok, wie "hardcode ich es dann ? in der anystores.js ?

**Comment by Tastaturberuf** [comment]
Schau dir dazu bitte die Google Maps API an, dort ist alles erklärt.

**Comment by sfx12** [comment]
ok, ich würd nur gern wissen in welcher Datei ich dass dann setzen muss ? weil wie gesagt im anderen thread.. reagiert die "map_google_dynamic.html5" mit keiner Änderung :-(

**Comment by Tastaturberuf** [comment]
Hatte ich geschrieben, in der anyStores.js im assets-Ordner. Danach aber
nicht mehr updaten, sonst sind die Änderungen weg.

**Comment by sfx12** [comment]
...läuft,

Danke

**Comment by Tastaturberuf** [comment]
Ich werde das JS demnächst als Template bereitstellen, so das es
updatesicher bearbeitet werden kann. Wird aber vor Mitte März nichts.
